ckanext/data_qld/templates/package/group_list.html

```diff
@@ -1,31 +0,0 @@
-{% extends "package/read_base.html" %}
```

Comment:
what was the reason we had this in the first place?

Reply:
I'm unsure, but it's been barely touched since it was added in 2019, and a diff with the core template showed no clear need for it.